src/welldevdemocontroller/io/Signup.java

```diff
@@ -0,0 +1,64 @@
+package welldevdemocontroller.io;
```

Comment:
welldevdemocontroller -> just "controller" would be fine. keep in mind naming is very important. you have to be careful about it. others must understand simply by seeing the name what its all about.